jbrowse/1.11.6/Dockerfile

```diff
@@ -0,0 +1,48 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         jbrowse
+# Software Version: 1.11.6
+# Description:      JBrowse is a fast, embeddable genome browser built completely with JavaScript and HTML5, with optional run-once data formatting tools written in Perl.
+# Website:          http://jbrowse.org/
+# Tags:             Genomics
+# Provides:         jbrowse 1.11.6
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/jbrowse 1.11.6/.
+# Pull Cmd:         docker pull biodckrdev/jbrowse
+# Run Cmd:          docker run --rm -p 8000:8000 biodckrdev/jbrowse
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)